cmd/state/main.go

```diff
@@ -85,7 +85,7 @@ func main() {
 	// Set up our legacy outputer
 	setPrinterColors(outFlags)
 
-	isInteractive := strings.ToLower(os.Getenv(constants.NonInteractive)) != "true" &&
+	isInteractive := strings.ToLower(os.Getenv(constants.NonInteractiveEnvVarName)) != "true" &&
```

Comment:
This is so I can test on macOS. Not intended for public use but don't care if someone figures out they can use this.